esmvalcore/preprocessor/_io.py

```diff
@@ -117,8 +117,8 @@ def load(file, callback=None):
             category=UserWarning,
             module='iris',
         )
-
         raw_cubes = iris.load_raw(file, callback=callback)
+    logger.debug("Done with loading")
```

Comment:
```suggestion
    logger.debug("Done with loading %s", file)
```
Would be more clear, especially when the log messages get mixed up because you're running with `max_parallel_tasks` > 1. Note that the preprocessor steps are already emitting log messages though, so as soon as you get the log message from the step coming after `load` (I believe it's `fix_metadata`) coming from the same process, you know that loading is done.

Reply:
Done